<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>764</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>65536</width>
    <height>65536</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1340</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="serverIpLabel1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>server1 ip:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverIpEdit1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>530</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.61</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverIpEdit2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>530</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>192.168.1.8</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverIpLable2">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>530</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>server2 ip:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverPortEdit1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>560</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>8090</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverPortLable1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>560</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverPortLable2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>560</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverPortEdit2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>560</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>8099</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectBtn1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectBtn2">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="snapBtn1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>550</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>snap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="snapBtn2">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>550</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>snap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="swapBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>660</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>swap</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLable1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgLable2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>500</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rmBtn1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>610</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rmBtn2">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>610</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imgLeft1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>630</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imgRight1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>630</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imgLeft2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>630</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imgRight2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>630</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="duEdit1">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>590</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recordBtn1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>580</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>record</string>
    </property>
   </widget>
   <widget class="QLabel" name="duLable1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>duration(s):</string>
    </property>
   </widget>
   <widget class="QLabel" name="duLable2">
    <property name="geometry">
     <rect>
      <x>1008</x>
      <y>590</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>duration(s):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recordBtn2">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>580</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>record</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="duEdit2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>590</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLable2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>660</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(136, 138, 133);
gridline-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard</set>
    </property>
   </widget>
   <widget class="QLabel" name="imgLable3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="statusLable1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(136, 138, 133);
gridline-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard</set>
    </property>
   </widget>
   <widget class="QPushButton" name="diffBtn1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>630</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>diff</string>
    </property>
   </widget>
   <widget class="QPushButton" name="diffBtn2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>630</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>diff</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>630</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1340</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
